src/V3Control.h

```diff
@@ -26,6 +26,32 @@
 #include "V3Mutex.h"
 
 //######################################################################
+struct LengthThenLexiographic final {
+    // Used to sort strings by length, then lexicographically
+    bool operator()(const string& a, const string& b) const {
+        if (a.length() != b.length()) return a.length() < b.length();
+        return a < b;
+    }
+};
+struct HookInsertEntry final {
+    int insID;
+    std::string insFunc;
```

Comment:
Please comment every struct/class member.

Add comment above as to struct's purpose.